### PR TITLE
Added some code examples.

### PR DESCRIPTION
**Pull request by SimoneStarace**

### CODE REVIEW

File: src/map/script_map.cpp [hunk 1]
```diff
@@ -166,9 +177,22 @@ static int CclCenterMap(lua_State *l)
...
 **  @param l  Lua state.
+**
+** Example:
+**
+** <div class="example">
+** 	<code>
+**		  -- Start view for player 0.<br>
+**		  SetStartView(0, 25, 12)<br>
+**		  -- Start view for player 1.<br>
+**		  SetStartView(0, 71, 38)
```
Comment (timfel) [inline src/map/script_map.cpp:193]: I think this should be `SetStartView(1, 71, 38)` :)

File: src/ui/script_ui.cpp [hunk 2]
```diff
@@ -221,9 +273,20 @@ static int CclSetVideoResolution(lua_State *l)
...
+**
 **  Get the video resolution.
 **
 **  @param l  Lua state.
+**
+** Example:
+**
+** <div class="example">
+** 	<code>
+**		resolution = GetVideoResolution()
```
Comment (timfel) [inline src/ui/script_ui.cpp:286]: here maybe this example would be better:
```
width,height = GetVideoResolution()
print("Resolution  is " .. width .. "x" .. height)
```

File: src/map/script_map.cpp (hunk 1 above)
Comment (SimoneStarace) [reply src/map/script_map.cpp]: https://github.com/Wargus/stratagus/commit/72f4bce9a18b577169f05e468e99e81cce150a7d#r52423732

File: src/ui/script_ui.cpp (hunk 2 above)
Comment (SimoneStarace) [reply src/ui/script_ui.cpp]: Fixed the GetVideoResolution example
https://github.com/Wargus/stratagus/commit/72f4bce9a18b577169f05e468e99e81cce150a7d#r52423776

Comment (SimoneStarace) [reply src/ui/script_ui.cpp]: Fixed the GetVideoResolution example
https://github.com/Wargus/stratagus/commit/72f4bce9a18b577169f05e468e99e81cce150a7d#r52423776

**Pull request merged**